File: submissions/sm_014_kondreddy/week_02/day_03/session_02/average_divisible_while.js
var i = 1;
sum = 0;
count = 0;
var avg ;
var num = 2;
while (i <= 10) {
    if (i % num == 0) {
        // console.log(num)
    sum = sum + i;
    count = count+1;
    }
    i = i +1;

}
avg = sum/count;
console.log(avg)

File: submissions/sm_115_suyash/week_13/day_4/cooding_1/average_divisible_while.py
print("Input diviser")
div_num = int(input())
print("Input max limit")
max_limit = int(input())
count = 0
avg = 0
sum = 0
i = 0
while i <= max_limit:
    if i % div_num == 0:
        count += 1
        sum += i
    i += 1
avg = sum / count
print(avg)
    